Frends.Avro.Serialize/Frends.Avro.Serialize/Definitions/Input.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Frends.Avro.Serialize.Definitions;
+
+/// <summary>
+/// Input parameters.
+/// </summary>
+public class Input
+{
+    /// <summary>
+    /// Input JSON string
+    /// </summary>
+    /// <example>{ "foo": "bar" }</example>
+    public string Json { get; init; }
```

Comment:
Please mention that this supports both single objects and also arrays of objects